src/main/java/Duke.java

```diff
@@ -1,3 +1,5 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
         String logo = " ____        _        \n"
```

Comment:
The printed logo, and the greeting message could be extracted and printed using a single method eg printIntroMsgs() so it appears as 1 line in main